frontend/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
what's the difference between this file and the manifest.json in the root directory? remove the useless one.
I believe manifest.json should be in the public folder

Reply:
Done